tsl/test/expected/bgw_reorder_drop_chunks.out

```diff
@@ -658,5 +658,5 @@ SELECT show_chunks('test_drop_chunks_table');
 (4 rows)
 
 --drop the view to allow drop chunks to work
-DROP VIEW tdc_view CASCADE;
+DROP VIEW tdc_view;
```

Comment:
I don't think we should keep the cascade notice below if we do not require `CASCADE`. These notices are typically displayed when there are dependent objects created by the user and `CASCADE` is required to remove them at the same time. 

However, the semantics we want is to treat the DROP as if it was acting on a single abstract object, so it makes no sense to keep the cascade notices about subobjects that are part of the abstract object.

Reply:
As we discussed, the message is something that PostgreSQL generates internally and setting up the dependencies turned out to not be trivial. I will skip this issue for this PR and investigate it separately.

Note that PostgreSQL has support for handling internal dependencies and allow objects to be part of an implementation as is the case for continuous aggregates. However, this requires a separate effort.